src/pages/instances/forms/CreateImageFromInstanceSnapshotForm.tsx

```diff
@@ -30,14 +31,17 @@ const CreateImageFromInstanceSnapshotForm: FC<Props> = ({
   const eventQueue = useEventQueue();
   const toastNotify = useToastNotification();
   const queryClient = useQueryClient();
+  const snapshotLabel = (
+    <ResourceLabel bold type="snapshot" value={snapshot.name} />
+  );
 
   const notifySuccess = () => {
     const created = (
       <Link to={`/ui/project/${instance.project}/images`}>created</Link>
```

Comment:
Shouldn't we use the image chip with its name here?

Reply:
Meaning a resource link for the image itself.

Reply:
I might be wrong since I don't have the complete context for this feature, looking at the code though I think this was done before because we don't have a way to get the image name at this point?

Reply:
Right, neither the form inputs nor the post response, nor the operation response contain the image description. Then let's keep it as simple as it is now.